index.js

```diff
@@ -1,57 +1,65 @@
 /* eslint-disable no-restricted-globals */
-let listOfBooks = [];
+// eslint-disable-next-line max-classes-per-file
+class Book {
+  static listOfBooks = JSON.parse(localStorage.getItem('books')) || [];
 
-function addBook(title, author) {
-  listOfBooks = JSON.parse(localStorage.getItem('books')) || [];
-  const serialNumber = listOfBooks.length + 1;
-  listOfBooks.push({ serialNumber, title, author });
-  localStorage.setItem('books', JSON.stringify(listOfBooks));
+  constructor({ title, author }) {
+    this.title = title;
+    this.author = author;
+    this.serialNumber = Book.listOfBooks.length + 1;
+    Book.listOfBooks.push(this);
+  }
 }
-// eslint-disable-next-line no-unused-vars
-function removeBook(event) {
-  const remSerial = event.target.parentElement.id;
+class Methods {
+  // constructor() {}
 
-  listOfBooks = JSON.parse(localStorage.getItem('books'));
-  // eslint-disable-next-line eqeqeq
-  listOfBooks = listOfBooks.filter((book) => book.serialNumber != remSerial);
+  static add({ title, author }) {
+    // eslint-disable-next-line no-new
+    new Book({ title, author });
+    localStorage.setItem('books', JSON.stringify(Book.listOfBooks));
+  }
```

Comment:
- [ ] Static method is directly called on the class. Static methods are that using them forfeits one of the advantages of objects. Objects are intended for data encapsulation. Instead of using the static method `add`, we can attach an array property with class and push parameters of the `add` method inside this property. And we return the array or we can get the updated array from the getter method.